release_notes/topics/18_0_0.adoc

```diff
@@ -4,6 +4,17 @@
 
 Recovery Codes as another way to do two-factor authentication is now available as a preview feature.
 
+== OpenID Connect Logout Improvements
+
+Few fixes and improvements were made to make sure that {project_name} is now fully compliant with all the OpenID Connect logout specifications:
+
+* OpenID Connect RP-Initiated Logout 1.0
+* OpenID Connect Front-Channel Logout 1.0
+* OpenID Connect Back-Channel Logout 1.0
+* OpenID Connect Session Management 1.0
+
+For more details, take a look at link:{adminguide_link}#_oidc-logout[{adminguide_name}].
```

Comment:
We are standardizing on  "For more details, see.."

Reply:
Ok, so I am changing it also for 3 other places in this file 18_0_0.adoc. Thanks